.NET/Microsoft.Recognizers.Text.Sequence/Chinese/Extractors/ChineseURLExtractorConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions;
+using Microsoft.Recognizers.Definitions.Chinese;
+
+namespace Microsoft.Recognizers.Text.Sequence.Chinese
+{
+    public class ChineseURLExtractorConfiguration : SequenceConfiguration, IURLExtractorConfiguration
+    {
+        public static readonly Regex IpUrlRegex = new Regex(BaseURL.IpUrlRegex, RegexOptions.Compiled);
+
+        public static readonly Regex UrlRegex = new Regex(BaseURL.UnicodeUrlRegex, RegexOptions.Compiled);
```

Comment:
BaseURL.UnicodeUrlRegex seems to be the only difference. Can't the config just have this and the base class get it from the config? Why do we need to repeat everything else?

Reply:
> BaseURL.UnicodeUrlRegex seems to be the only difference. Can't the config just have this and the base class get it from the config? Why do we need to repeat everything else?

I am not sure if there will be other special circumstances, so I kept this part first.

Reply:
So let's make the change for the config to only have the difference, ok?